projects/azure/rbac.yaml

```diff
@@ -3,6 +3,13 @@ apiVersion: v1
 kind: List
 items:
 
+- kind: ProjectRequest
```

Comment:
Move this alongside the rest of the project requests in https://github.com/openshift/release/blob/master/cluster/ci/config/roles.yaml

Reply:
@smarterclayton @stevekuznetsov do you guys prefer a monolithic config with all the policies of the CI cluster or each team maintaining its own config?

Reply:
no strong feelings as long as makefile works